src/components/filters/DateFilter/index.tsx

```diff
@@ -90,13 +90,13 @@ export const DateFilter: React.FC<Props> = (props) => {
 
   const emptyValues = React.useMemo(() => !from && !to, [from, to])
   const validDates = React.useMemo(() => {
-    if (fromDate && fromDate.current && toDate && toDate.current) {
+    if (from && to && fromDate && fromDate.current && toDate && toDate.current) {
       return toDate.current.checkValidity() && fromDate.current.checkValidity()
     } else {
       return true
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [from, to])
+  }, [from, to, fromDate, toDate])
 
```

Comment:
Everything seems to work smoothly, great job!.
I'm not sure if this is causing the following problem with validation:

![Peek 22-10-2020 15-57](https://user-images.githubusercontent.com/358059/96918125-36a3b580-1480-11eb-8a00-34f5e4c1263c.gif)

Remember that `from` and  `to` are the controlled values, they are in the dependency list to force checking the refs that don't cause re-renderings but allow to call the method of `HTMLInputElement`.

Reply:
I added the FROM and TO because  if the inputs are empty, was still checking and display the warning as Elena says in a previous comment

![Screenshot_20201022_161017](https://user-images.githubusercontent.com/1144028/96918860-54bde580-1481-11eb-8585-caff4236c736.png)

Also, I think is not related to the purpose of this PR, a PR already merged introduced this problem, I just try to give it a shot to solve it

Reply:
@mariano-aguero This isn't happening in https://conditional-tokens-explorer.dev.gnosisdev.com/#/positions

Reply:
> @mariano-aguero This isn't happening in https://conditional-tokens-explorer.dev.gnosisdev.com/#/positions

This is a problem related to the DateFilter component, so I will remove the few lines that I changed, and will comment to Elena to recheck this, and in case the problem still exist, create and issue